pombola/core/tests/test_organisation_histories.py

```diff
@@ -0,0 +1,33 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+from pombola.core import models
+
+
+class OrganisationHistoryTest(TestCase):
```

Comment:
This isn't really a helpful test. All this is testing is that you have a django model, and django does indeed do its job. This model is so simple, this test is pretty much just testing that django works the way they say it does, so I'd prefer not to have a test like this wihch someone will have to maintain, for little benefit

Reply:
@kaysiz why did you unresolve this?